### DISCUSSION TITLE
기획안 브레인 스토밍 및 정리

### DISCUSSION THREAD

**Discussion by aespablo** [envybros/nomads-planet#4] [Announcements]
여기서 새로운 아이디어나 기술 검토 등 많은 내용들을 작성한다.

**Comment by aespablo** [comment]
### 콘텐츠 명

> ENVY'S WONDER WORLD

### 타겟층 및 타겟 대상

> 놀이공원에 가고 싶은 사람들 전부
> 도전적인 20대 패셔니스타 인플루언서 (슈퍼 E)

### 플레이어 캐릭터

> `MetaQuest Avatar SDK` 검토해서 어느 정도 가용할 수 있을지 체크해야함
> 그런 캐릭터면 꽤 편하게 이용할 수 있을 것 같아 보인다.

### 주요 컨텐츠 

> 놀이기구
> 귀신의 집
> 사람들과 소통 (VR-Chat 느낌, `MetaQuest SDK` <- 검토

**Comment by aespablo** [comment]
### 도우미 캐릭터

> 튜토리얼부터 해서 여러 도움말을 알려줄 캐릭터가 하나 필요하다.
> 이 캐릭터 이름을 `ENVY BOT` 이라 지으면 될 것 같다.

**Comment by aespablo** [comment]
# 공간 요소 및 어트랙션 정리

## 놀이공원 필드

0. 폐 행성 (조그만 섬)에 놀이공원을 만들었다고 생각하면 된다.
1. 곳곳에 위치한 나무들과 NPC들
2. 지나다니는 플레이어들
3. 뜬금없이 위치한 감성 피아노 (리듬게임?)
4. 온갖 미디어 아트에서 얻은 아이디어들의 향연

> 사람들끼리 대화도 가능하고, 파티도 가능함 (셀카도 가능?)
> 파티원끼리 같이 씬을 넘어갈 수 있음
> 랜덤 매치된 사람들과 함께 (최대 인원수에 맞게) 놀이기구를 탈 수 있다.
> 각 어트랙션들은 포탈로 되어있으며, 앞에는 안내문이 들어있다.
> 어트랙션에 대한 설명은 클릭시 UI로 볼 수 있다.
> 그 놀이기구에 걸맞는 입구를 만들어두고, NPC를 배치한다.

## 롤러코스터 (AKA 로켓코스터)

1. 무중력
2. 우주까지 올라갔다가, 태양도 보고 바다 속으로 들어가기도 함
3. 물고기들도 보고 그러고 다시 도시 위를 지나
4. 원래 곳으로 돌아옴. 플레이 시간은 3분 정도

> 쿨타임 5분
> 하늘로 올라가면서 롤러코스터 레일 벗어나거나 로켓처럼 바껴도 재미있을듯
> 미디어 아트의 개념을 여기 입히기 (어려울 시 영상 재생)
> 중간에 하차 가능
> 달 세계 여행 오마주

## 회전목마

1. 원래는 배경에 군중들이 위치해 있다.
2. 그러다가 돌면 돌수록 약간 옛날 영화느낌으로 말도 달리다가~
3. 아무튼 여기도 미디어 아트 입히자.
4. 그렇게 스토리가 다 끝나면, 다시 서서히 흐려지다가 멈춘다.

> 미디어 아트 아이디어 생각하기 좀 어려울듯
> 한국 관련된 이야기가 있으면 더 재미있을 것 같다.

## 동물원

1. 말이 동물원이지, 사실상 동물들과 상호작용하는 곳
2. 신비한 동물들이 가득해야함

## 아쿠아리움

1. 여긴 거대 공룡 상어 한마리가 애교부리는 곳
2. 쥬라기공원 모티브
3. 가끔 이벤트도 함 (시간 정해두기,, 2시간 간격으로?)
4. 거대 보스 레이드하는 느낌을 내야함
6. 웅장한 느낌

> 여긴 그냥 묘기 보는 곳
> 말이 아쿠아리움이지,, 메가로돈 보러가는 곳이다.

## 텔레포트 (이동 수단)

1. 텔레포트 이동을 뿅! 하고 가면 안된다.
2. 화면 서서히 효과 들어가면서, 그렇게 이동해야 한다.
3. 등장 또한 마찬가지
4. VR에 적합하게 만들자.

---

... 충분해보이지...?

**Reply by aespablo** [reply]
이제 각 요소별 에셋들이랑 월드를 준비하면 된다,,,

**Comment by pulppixel** [comment]
이거 기획안 대폭 변경됨...

**Comment by pulppixel** [comment]
이거 처음에 로그인창을 VR에서 보면 어떨지 확인해봐야해

**Comment by pulppixel** [comment]
이거 의심 안해도 될 것 같아

아이디어 정말 좋아

**Comment by aespablo** [comment]
[Metaverse Shopping Center](https://www.behance.net/gallery/144666665/D-05-Metaverse-Shopping-Center)

위 링크 타고 가서 좀 아이디어를 얻어야 할 것 같다.

비록 메인 화면에서는 차 안에 있지만,,, UI 개발을 끝장나게 해서, UI 상에서도 충분히 콘텐츠들을 즐길 수 있도록 해야 한다....

창 밖에서도 재미난 일들을 벌여야,,

**Comment by aespablo** [comment]
차 모델

1. [Aston Martin DB4GT Zagato
](https://www.cgtrader.com/free-3d-models/car/luxury-car/aston-martin-db4gt-zagato-1960-1963)
2. [Aston Martin Vantage](https://www.cgtrader.com/free-3d-models/car/sport-car/aston-martin-v12-vantage-s-2016-with-hq-interior-rigged)
3. [Supercar Opened Up](https://www.cgtrader.com/free-3d-models/car/sport-car/supercar-opened-up)
4. [Luxury car Porshe 911](https://www.cgtrader.com/free-3d-models/car/luxury-car/luxury-car-porshe-911-carrera-4s-based-on-a-real-car)
5. [Porsche Carrera Tuned](https://www.cgtrader.com/free-3d-models/car/sport-car/personaliced-porsche-carrera)

**Comment by pulppixel** [comment]
### P2E

### 지갑 시스템

딱히 니모닉을 통해 지갑을 연동할 필요는 없음
그냥 **주소 해시값** 하나만 연동할 수 있도록 하기. (변경도 가능)

하지만!!! 이거 중복은 안됨. 닉네임하고 똑같이 가면 될듯

그리고 **Web3 Faucet**하고 기능 완전히 똑같이 가면 될 것 같은데?
운전 다 끝나면, 유저가 등록해둔 해시 값으로 전송해주기 (버튼 클릭 必)

**Comment by aespablo** [comment]
## 모바일 앱

- 스테픈 벤치마킹
- 네비게이션 구현 후, 이용시 토큰 지급
- 더 비싼 차 NFT가 있으면 더 많은 토큰 지급

**Comment by pulppixel** [comment]
[자동차 구동 원리](https://kids.hyundai.com/kidshyundai/AutomobileManagement/learnauto/automobileDet.kids?ctgrMgrpCd=&cotnSn=3304#:~:text=%EC%9E%90%EB%8F%99%EC%B0%A8%EC%9D%98%20%EB%B0%94%ED%80%B4%EB%8A%94%20%ED%97%88%EB%B8%8C,%EB%AF%B8%EB%8A%94%20%ED%9E%98%EC%9D%B4%20%EB%B0%9C%EC%83%9D%ED%95%A9%EB%8B%88%EB%8B%A4.)

내가 이런 것도 다 찾아보다니,,
